### PR TITLE
UMass OSPool

### PR DESCRIPTION
**Pull request by rafaellopesdesa**
Adding UMass OSG SITE at UMass Amherst

### CODE REVIEW

**Review by matyasselmeci — APPROVED** [review]

**Review by brianhlin — CHANGES_REQUESTED** [review]
Let's call the site/resource group `UMass OSPool`

**Comment by rafaellopesdesa** [comment]
> Let's call the site/resource group `UMass OSPool`

Ok, I will update it.

**Review by brianhlin — APPROVED** [review]
LGTM

File: topology/University of Massachusetts - Amherst/UMass OSPool/UMass-OSPool.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        ID: OSG1000284
+        Name: Eduardo Bach
+      Secondary:
+        ID: OSG1000443
+        Name: William Leight
+  Description: Direct provisioning of the NET2 Kubernetes cluster for OSPool, in the osg-opportunistic namespace
+  FQDN: provisioner-osg.compute.net2.mghpcc.org
+  Services:
+    Execution Endpoint:
+      Description: Direct provisioning of the NET2 Kubernetes cluster for OSPool, in the osg-opportunistic namespace
```
Comment (brianhlin) [inline topology/University of Massachusetts - Amherst/UMass OSPool/UMass-OSPool.yaml:22]:
```suggestion
---
GroupDescription: OSPool resources at UMass Amherst
Production: true
SupportCenter: USATLAS
Resources:
  UMass-OSPool-Provisioner:
    Active: true
    ContactLists:
      Administrative Contact:
        Primary:
          ID: OSG1000284
          Name: Eduardo Bach
        Secondary:
          ID: OSG1000443
          Name: William Leight
      Security Contact:
        Primary:
          ID: OSG1000284
          Name: Eduardo Bach
        Secondary:
          ID: OSG1000443
          Name: William Leight
    Description: Direct provisioning of the NET2 Kubernetes cluster for OSPool, in the osg-opportunistic namespace
    FQDN: provisioner-osg.compute.net2.mghpcc.org
    Services:
      Execution Endpoint:
        Description: Direct provisioning of the NET2 Kubernetes cluster for OSPool, in the osg-opportunistic namespace
```

Fixup formatting

**Pull request merged**